Specs/DateTime/German/DateTimeExtractor.json

```diff
@@ -36,7 +36,7 @@
     ]
   },
   {
-    "Input": "Ich gehe am 15. November um 8 Uhr weg.",
+    "Input": "Ich gehe am 15. November um 8 Uhr los.",
```

Comment:
I've seen "8 Uhr weg" and "8-Uhr weg" before. Aren't they also common/informal/regional/etc?

Reply:
Both are correct, both mean slightly different things. I can remember why it was changed but it should make a difference from an extraction standpoint. Both are not the correct translations for the English term in the spec anyway. ;)